lua/starfall/libs_sv/wire.lua

```diff
@@ -632,6 +632,17 @@ function wirelink_methods:isValid()
 	return wlunwrap(self) and true or false
 end
 
+--- Returns current state of the specified input
+-- @param string name Input name
+-- @return any Input value
+function wirelink_methods:inputValue(name)
+	local wl = wlunwrap(self)
+	if not wl then return end
```

Comment:
unwrap shouldn't return nil, it will error instead

Reply:
It will return NULL Entity.
```lua
local ent = prop.create(chip():getPos(), chip():getAngles(), "models/holograms/cube.mdl", true)
local wl = ent:getWirelink()
ent:remove()

timer.simple(0.1, function()
    print(wl:entity()) --> [NULL Entity]
    print(wl:isValid()) --> true
end)
```
How about implementing `getwl` (similar to `getent`)?
My latest commit should work assuming that `wlunwrap` can only return `NULL Entity` and never `nil`.